scipy/spatial/src/distance_impl.h

```diff
@@ -630,7 +628,7 @@ pdist_minkowski(const double *X, double *dm, npy_intp num_rows,
     if (p == 2.0) {
         return pdist_euclidean_double(X, dm, num_rows, num_cols);
     }
-    if (sc_isinf(p)) {
+    if (isinf(p)) {
```

Comment:
Here there's no header at the top anymore, so it's not clear to me where this `isinf` is being picked up from. Perhaps add `from libc.math cimport isinf` at the top?

Reply:
Good point, it should be `#include <math.h>`, will add.